### DISCUSSION TITLE
Balance Incontinence Risk/Reward

### DISCUSSION THREAD

**Discussion by zippity21** [zippity21/Stardew_Valley_Regression_Mod#26] [Ideas]
Given the target community, it is expected that most will try to become incontinent as quickly as possible. This is likely to make the process of slowly losing continence feel like a mere inconvenience.

I recommend making more consequences for losing continence, to give players an incentive to stay dry. The challenge of this mod is to balance hunger, thirst, time, energy, and the need to pee. So, if there is no punishment for not using the big-kid potty, why would anyone try to take it into consideration?

**Comment by ghost** [comment]
I feel that this request discounts the player-base that isn't playing ABDL mods for increased difficulty. Some people just want to roleplay as a diapered Stardew character. Others (myself included on this one) may just want to roleplay going through gradually declining continence in a lifesim-style game and enjoy the day-to-day diaper management aspects.

Not saying that there shouldn't necessarily be applicable in-game mechanics regarding diapers (bed drying, cost of diapers, social consequences, and speed and defense decreases for blowouts and wet leaks all being good examples), but I worry that putting too many or too harsh of consequences on diapers and accidents will only make the game more of a chore to play, rather than enhancing it.

**Comment by zippity21** [comment]
I suppose that's true. I put it here more as a potential suggestion. It probably belongs in the discussion section, but I hadn't set it up yet.

Perhaps not adding more consequences, but maybe just better balancing the existing ones, like:

- Stamina hit per wake-up at night
- Bed/cloth drying times based on messiness
- Duration and magnitude of debuffs
- Magnitude of social repercussions (both positive and negative)
- Cost of underwear
- Bladder, Bowel, Hunger and Thirst fill rates
- Rate of continence loss
- Effects of food & drink

I think you're right that there is enough there. I just don't know if things are as balanced as they should/could be. But I really need community feedback. So I think this is an open discussion, and I don't think I'll make any changes in the near future.

**Reply by ghost** [reply]
Woo, discussion! So, opinions on the current systems:


- Stamina hit per wake-up at night - This is fine where it's at, I feel. Noticeable, but not TOO impactful. Good reminder to go potty or change diapers before going to sleep

- Bed/cloth drying times based on messiness - Same as above, working as intended imo.

- Duration and magnitude of debuffs - I think these are good where they're at as well. The speed debuff in particular hurts and adds a very real consequence to going into the mines or doing some late night fishing unprepared.

- Magnitude of social repercussions (both positive and negative) - We've discussed this before, but I feel some major changes could be made here for the better. Without diving into the creation of an entirely new stat and affection system, however, I think the current levels are alright for what they are. One thing in this arena I'd like to see is NPC reactions to outwardly apparent wet/messy leaks when you interact with them. Wet pants in particular should be immediately noticeable. Repercussions (disgust)/benefits (sympathy) shouldn't be as heavy as NPCs noticing you toileting in public or using your pants/diaper, but should still be present. It feels weird going into the general store with a leaky diaper or wet pants and everyone acts like it's business as usual.

- Cost of underwear - I think this could go either way. If you want the mod to low-key _try_ to wear the player's continence down and put them back in diapers (and I feel this is kind of implied by the initial note that Jodi sends about the area having certain strange effects on people), adult and dino undies should probably be more expensive than they are currently. This way it takes longer to build up a stock and you have to weigh purchasing undies vs. other resources you need in your first year. This makes diapers a much more tangible risk to end up in since you have a large stock of them right from the start and they could easily end up being your only option after an accident or two. Might also be good to introduce a limited lifetime on undies, so that they become ruined and need replacing after too many accidents. Say, 5 or 10 or so.

If the game _isn't_ trying to push the player towards incontinence, I think prices are good where they're at. Undies are a viable choice from the beginning and diapers are a noticeable ding in monthly income if you're buying disposables regularly, or a significant one-time expense if going cloth.

- Bladder, Bowel, Hunger and Thirst fill rates - I'm in a weird place with this most recent update. I think hunger and thirst are A+ right now. You could maybe tune them to advance a _tiny_ bit more quickly if you want to make them a more pressing need. As it stands I'll only really need to eat and drink once per day, if that. Maybe twice if I'm going really hard on wood-cutting or mining. Bladder and bowel fill rates, however...My character is having accidents much more often with this update. Maybe just due to my own inattention? Not sure if I just began to filter out all of the notices or if the fill rates are advancing much more quickly than the previous version, but figured it was worth mentioning. Previous version I could go full weeks without accidents. This version I'm lucky if I make it two days. I only just started using debug mode for the new file, so unfortunately I can't compare any hard numbers very well to last version.

- Rate of continence loss - It seems to be faster this version, but that may just be due to the above-mentioned accident frequency. Regardless, I feel it's in a good place currently. Again, the initial message from Jodi notes the strange effects of the area, so I feel it's an overall benefit to have potty problems be a noticeably progressing event that happens over time (as friendship does).

- Effects of food & drink - In a perfect place currently, imo. Being penalized so harshly in the previous version for eating small items like seaweed and algae when energy was drastically needed was brutal. I think you've hit a good median with this update.

**Reply by zippity21** [reply]
I'm glad the system overhaul sounds like it was worth it!

To comment on the things that sound like they need improvement:

- Social repercussions
I agree. Without a whole new system, it's probably fine with where its at. Though I definitely feel like there should be a difference between clean, wet, messy, and overflowing with how NPCs respond. I do hope to one day add such a system, but I feel like there are higher priority issues at hand, and lower hanging fruit.

- Cost of underwear
I agree with the sentiment that anything washable should also have a durability, so there isn't a one-and-done. The durability needs to be enough so that something like a cloth diaper investment makes sense vs a bunch of disposables (cloth already has the downside of only being able to be worn once a day). Funny enough, as I've said before, I haven't really played this game, so I don't know what is considered a high or low cost.

- Fill rates
These can be adjusted rather easily now due to the code refactoring I did. And now that you bring it up, the hunger and thirst _are_ going slower than intended. A full belly should last about a full day without doing a lot of work (work should make you more hungry), but I reduced the loss at night to 1/3 of daytime (less activity + I didn't want to be **that** evil with night time accidents), But I didn't add the corresponding rate increase to the daytime to compensate.

As for the bladder/bowel filling messages, they ***are*** coming up less frequently and it is based on your current continence (the less continent you are, the less likely it is that the warning message will show up). Though I think the scale needs to be adjusted a bit since it does seems they become too infrequent too quickly. (I don't think a major adjustment, just a fine tuning of the scale).

I believe the fill rates for bladder/bowels are faster since I purposefully scaled it so from a full belly (without any additional eating/drinking) and with full continence you need to go pee ~4.5 times a day and poo ~2. This becomes more frequent as your bladder and bowels can hold less as you lose continence. Though, on the plus side, the diapers should still last just as long since the total amount over the day is still the same.

As for how often you eat: Are you on easy mode? Just making sure, since in hard mode you should be more hungry/thirsty after waking up, but wake up with a full belly in easy mode. And where a full belly is at 4500 points, and most food only adds 200, I'd suspect you'd need to eat a lot more often. I just want to make sure this is true.

**Reply by TikkuTokku** [reply]
I would like to add my two cents on this.

I feel it's fair to have a concern that we'd speedrun to diapers and then do everything the mod has to offer and get bored fast. I know I would be hooked for a LONG time through just exploring the new ways characters interact with your potty troubled character, which was a big motivation for me to start working on that.

**Reply by ghost** [reply]
> As for how often you eat: Are you on easy mode? Just making sure, since in hard mode you should be more hungry/thirsty after waking up, but wake up with a full belly in easy mode. And where a full belly is at 4500 points, and most food only adds 200, I'd suspect you'd need to eat a lot more often. I just want to make sure this is true.

Sorry for my super late reply! Put down Stardew for a bit to play some Gungeon and completely spaced the conversations here x.x Anyways, to answer your question, I had been on easy mode at the start of my playthrough, but had switched in summer of year 1. From what you're describing though, it does definitely sound like easy mode's hunger/thirst and fill rates were in effect. What I don't get though is how I was seeing the stamina drains from wakeups and bedwetting/messing, but not the other aspects of normal mode. I guess from changing "difficulty" mid-playthrough?

Saw you pushed a new update btw! Gonna patch and give it a shot over my weekend :3 Cheers!

Edit: Oh, and regarding this!

> Funny enough, as I've said before, I haven't really played this game, so I don't know what is considered a high or low cost.

It really varies by player. People new to Stardew/farming sims will struggle for the first year with finding the extra 1000g to pick up a "pack" of 10 disposable diapers per week from Pierre's. The first year as a fresh player basically consists of frantically running around trying to balance making enough money to buy seeds, farm buildings, and food (especially with the mod's enhanced emphasis on food/drink, I'd imagine) and do enough other stuff to complete Community Center bundles.

Things level off a TON in the 2nd year though once they get better at farm automation, build up natural daily income with farm animals, and goals get checked off from the list. It's genuinely insane how much more efficient a new player's 2nd year is, let alone their 2nd playthrough of year 1.

Experienced players will barely feel the dent in income. There are guides out there that detail how to casually make 5 million G in your first year, for instance. I don't really think you should balance for this, but it might be useful to know the upper limits of what's possible.

I suppose it depends on which audience you want to cater to. I'd hazard a guess that the majority of the mod's playerbase are probably somewhat experienced SDV players, maybe having gone through a couple runs in the past (this was me, for instance). That said, I do see a decent amount of new players picking it up as well just from seeing it mentioned on FTT and Reddit, so it's really up to you. I don't think anyone would fault you for going either way.

**Reply by ghost** [reply]
So quick update on current balance opinion for 1.2.4. Started a new file again and I like where everything is at except for hunger. The first spring in particular is really rough when you need to chow down on 10 of any food item just to fill the hunger bar once. I like the idea of different foods being more filling, but early on you pretty much only have the initial spring onions, whatever you can grow fast (parsnips and potatoes, though they're generally meant to be part of your early income generation), and the very little that can be afforded from the saloon.

3 or 5 units of food for a full hunger bar might feel a bit better, or perhaps just setting spring onions and parsnips to have higher hunger-fill values so the early weeks aren't quite so rough.

**Reply by zippity21** [reply]
Speaking of late replies, I guess I didn't see this from 3 weeks ago, apologies.

Given your explanation of relative difficulty of the first year, and previous discussion about the intended 'difficulty' of this mod; I think it'd be more than fair to reduce how many things you need to eat to fill up. Five sounds about right.

As for early game, do you think increasing the starting gift from Jodi would be enough?

The other thought that I have would be maybe get a recurring care package? Or maybe you could go to the doctor for a free change into a low-end diaper once a day? Just thoughts. I'd love to hear yours.

**Reply by ghost** [reply]
Hey, no worries! I think 5 should be a really good value change. That alone may actually solve all the early-game issues. If the player is eating less, they're having accidents less/filling diapers less frequently, so I think the starting care package is probably good where it's at.

Just as an example, for this run I've been a bit more haphazard about paying attention to the warning notifications (seems to fit well with the new "grandpa told me about your little problem" exposition in the initial care package :p) and stuck with just the default undies and care package. Even without paying a lot of attention, I didn't run out of pull-ups until ~week 3 spring and still haven't even hit half of the paw-print pamps by week 2 of summer. I already have enough income to be able to afford disposables or to start looking into cloth, but a fresh player might need a few extra weeks, so I think the initial 40 pull-ups/40 diapers is already in a good spot for experienced players, while the hunger changes will make it a lot more accessible for new players.

That being said and balance aside, I love the idea of Harvey offering a free change per day at the clinic! Or at least there being some sort of interaction there. The player having a continence issue seems like something he, as the town doctor, would likely end up knowing about. Especially after the social event that happens with him at 4 hearts where he gives you your annual checkup. The mod currently lacks any sort of daily ABDL-related events outside of diaper-use, so I think a lot of immersion could be gained from a regular interaction like this.

Perhaps Harvey's 4-heart event could unlock the daily diaper change perk? Though it's generally more convenient to self-change in the game and the player should be able to maintain a stock of diapers by the time that event hits, so maybe as an incentive to go to him daily he also uses "premium" medical diapers for the changes? Something akin to the real-life Megamax, a medical diaper that has top-tier absorbency like the recently-added heart diaper. They could be something exclusively received through changes until the player hits 6 hearts with Harvey, at which point they become available through the clinic store (At 6 hearts Harvey asks the player to keep his participation in the town aerobics class a secret, so opening up sales of the medical diaper would be a sensible in-character reward for the player agreeing to do so). Or 8 or 10 hearts, though I feel those would be less fitting for the character. Or none of these at all, haha, just spitballing here!

Sorry if this is a bit too far off from what you were thinking. I have a bad habit of taking ideas and running with them x3

**Reply by TikkuTokku** [reply]
I haven't had too much of a problem with that, but I fish a lot so I've generally got plenty of food.

Do higher quality items increase hunger more? That might help things along, especially with fishing and finding the spring onions in the South forest.

**Reply by zippity21** [reply]
I think that @LilBearBlue has a point though. I'd rather lean on the side of easier than harder since I feel that this mod is more about the ABDL stuff than survival.

I'd like to make different foods have different effects, but that is harder than with beverages since they are organized differently.

**Reply by ghost** [reply]
> I haven't had too much of a problem with that, but I fish a lot so I've generally got plenty of food.

I fish a lot in the first year too, maybe a 60-30-10 split between fishing, mining, and lumber/foraging, but I think the difference might be that I sell all fish x3 Particularly once Fishing 5 gets hit and their value increases, eating 10 in the morning to fill my hunger bar and watching all that cash drain away wouldn't feel so good. 

You make a good point that my problem is one that can be fixed via a different playstyle, however. I could divert the cheaper fish to food instead of cash. Honestly though, my issue isn't just with the additional difficulty. I've made things work by aggressively hoarding regular quality crops, forageables, algae, and seaweed XD So even without adding some fish in it's still workable in the current state (though I do think newer players would struggle to make money in their first year as-is).

My problem with it is more that eating 10 of something to get full doesn't make a ton of practical sense. Even with something fairly light like kale or spring onions, how many could someone eat before actually getting full? Not that SDV is the most irl-accurate sim game out there or anything, but so long as all food provides equal hunger, I think eating 5 things for a "meal" feels a bit more true/immersive than 10, if that makes sense.

**Comment by zippity21** [comment]
I figured out how to convert it to a discussion. Hurray!

**Comment by Gooberology** [comment]
I've been thinking about this too. What ive thought, is that having a high enough incontinence to confidently wear proper underwear should give minor buffs, like maybe slight speed from the lack of diaper waddle, meanwhile for low continence, being in a clean diaper should be encouraged with some kind of lesser buff. That way being potty trained has rewards that makes it worth considering, while people who want to still be in diapers are rewarded at least a little for staying clean too.
